docker-local-mojaloop/docker-compose.yml

```diff
@@ -0,0 +1,424 @@
+version: "3.7"
+
+networks:
+  mojaloop-net:
+    name: mojaloop-net
+
+services:
+  central-ledger:
+    image: mojaloop/central-ledger:v9.5.2
+    container_name: central-ledger
+    command:
+      - "sh"
+      - "-c"
+      - "sh /opt/wait-for/wait-for-central-ledger.sh && node src/api/index.js"
+    ports:
+      - "3001:3001"
+    volumes:
+       - ./docker/central-ledger/default.json:/opt/central-ledger/config/default.json
+       - ./docker/wait-for:/opt/wait-for
+    environment:
+      - LOG_LEVEL=info
+      - CLEDG_SIDECAR__DISABLED=true
+      - CLEDG_MONGODB__DISABLED=false
+    networks:
+      - mojaloop-net
+    depends_on:
+      - mysql
+      - kafka
+    healthcheck:
+      test: ["CMD", "bash", "-c" ,"apk --no-cache add curl", ";", "curl", "http://172.17.0.1:3001/health"]
+      timeout: 20s
+      retries: 10
+      interval: 30s
+    extra_hosts:
```

Comment:
What's the need for these extra hosts here? I only ask as I haven't played around this feature in docker-compose at all

Reply:
I believe this is a linux only issue.

So. Docker containers can't resolve hostnames using `/etc/hosts` on a person's local machine in linux. I need to add the hosts so the services can talk to each other.

While on macOs I believe it works out of box because of DNS caching. Hence why I believe this bug went unnoticed maybe?

https://stackoverflow.com/questions/43471292/why-docker-container-doesnt-resolve-hosts-as-defined-in-etc-hosts-on-host-machi

This is sort of a brute force fix(?). It is the plan to investigate a cleaner solution.

Reply:
Cool. Good to know. Let's create a follow up story.

Another option could be to use host based networking with docker, but that opens up another can of worms.